geoportailv3/models.py

```diff
@@ -17,6 +17,10 @@
 Base = sqlahelper.get_base()
 
 
+LayerWMTS.__acl__ = [DENY_ALL]
+WMTSDimension.__acl__ = [DENY_ALL]
```

Comment:
I think that we should do this for the ``LayerInternalWMS`` and ``LayerExternalWMS``, but not for the ``LayerWMTS`` and ``WMTSDimension``. The only consequence of that is that the table will be display or not in the admin interface.